pkg/kv/store_message.go

```diff
@@ -0,0 +1,63 @@
+package kv
+
+import (
+	"context"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type StoreMessage struct {
```

Comment:
Please add documentation for this Store. Not sure I understand what it does and why we need it.